### ISSUE TITLE
Unable to focus “go to top” with keyboard

### ISSUE THREAD

**Issue by ZoeBijl** [nikkilr88/nikkilr88.github.io#10]
The “go to top” clickable `div` that appears when you scroll down a little cannot be focused with a keyboard.

https://github.com/nikkilr88/nikkilr88.github.io/blob/8d5d1790fe9081f411b99bd9322c932acfc722e7/index.html#L368

This could be solved by adding `tabindex="0"`. That will allow keyboard users to focus the `div`. But making a `div` focusable is generally a bad idea. If we find ourselves at the point of making a `div` focusable we should take a step back and ask ourselves two questions “what is this thing?’ and “what does it do?” In this case we can answer those with “a link” and “scroll the page to the top”.

The code for this could look something like:

```HTML
<body id="top">
…
<a href="#top">
  <i class="fas fa-arrow-up" aria-hidden="true"></i>
  <span class="visually-hidden">Go to top</span>
</a>
```

The `aria-hidden` makes sure the icon isn’t announced to assistive technologies (such as screen readers and Braille readers). The `span` with the `visually-hidden` class provides the link with an accessible name—which is used by assistive technologies.

The CSS for that class should look like this:

```CSS
.visually-hidden {
  position: absolute;
  width: 1px;
  height: 1px;
  overflow: hidden;
  clip: rect(0,0,0,0);
  clip-path: inset(50%);
  white-space: nowrap;

}
```
